src/App/ImportUshahidiV2/Mappers/UserMapper.php

```diff
@@ -14,7 +15,7 @@ class UserMapper
         'member'     => 'user'
     ];
 
-    public function __invoke(array $input) : Entity
+    public function __invoke(int $importId, array $input) : Entity
```

Comment:
Unless we make it part of a generic object it will be difficult to ignore it. The reality is, I think, that given the shift in data model this is required but in future data models we should be covered by migrations.

Reply:
... I didn't follow that at all. Did you mean that in future it would be tracked as part of a model itself or something?